Essentials/src/com/earth2me/essentials/commands/Commandworth.java

```diff
@@ -84,7 +84,7 @@ private BigDecimal itemWorth(CommandSource sender, User user, ItemStack is, Stri
             amount = ess.getWorth().getAmount(ess, user, is, args, true);
         }
 
-        BigDecimal worth = ess.getWorth().getPrice(ess, is);
+        BigDecimal worth = ess.getWorth().getSellPrice(ess, is);
```

Comment:
Should `/worth` list both the buying and selling prices?

Reply:
There should probably be a way to configure whether or not it shows up, in the case that servers don't want to use both buy and sell at the same time, for example.